unity-renderer/Assets/Scripts/MainScripts/DCL/CRDTProtocol/Executor/CRDTExecutor.cs

```diff
@@ -26,6 +26,9 @@ public CRDTExecutor(IParcelScene scene, ECSComponentsManager componentsManager)
 
         public void Dispose()
         {
+            if (disposed)
+                return;
```

Comment:
We should assert above that we are not doing any operation if `disposed` (calling `Assert` so the  comprobation doesn't occur on production).